contracts/libraries/Events.sol

```diff
@@ -465,4 +465,20 @@ library Events {
         bool[] approved,
         uint256 timestamp
     );
+
+    /**
+     * @dev Emitted when the user wants to enable or disable the follow.
+     *
+     * @param profileId The token ID of the profile to which this followNFT is associated.
+     * @param owner The profile owner who executed the troggle.
+     * @param enabled. Boolean value that enable / disable follow.
```

Comment:
`.` not needed
Would change the description to `Whether the FollowNFT is enabled/disabled.` for consistency